package.json

```diff
@@ -129,7 +129,7 @@
     "prettier": "2.3.2",
     "print-js": "1.6.0",
     "raw-loader": "4.0.2",
-    "remark-cli": "9.0.0",
+    "remark-cli": "10.0.0",
```

Comment:
This might be breaking but I'm not 100% sure yet. I've just been adding it to exceptions.

Reply:
added